### PR TITLE
Add improvements to the test and package guides

### PR DESCRIPTION
**Pull request by Dilhasha**
## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md [hunk 1]
```diff
@@ -125,3 +125,24 @@ By default, running the `bal semver` command on the root directory of the packag
...
+
+To deprecate a particular version of a package on Ballerina Central, the package owner can run the following command. 
+An optional deprecation message can also be included, which will be displayed to current users of the package.
+
+```
+$ bal deprecate <org-name>/<package-name>:<version> --message <deprecation-message>
+```
+
+A deprecated package version will not appear in package searches on Ballerina Central or the CLI. Additionally, it will not be be used for dependency 
+resolution unless it is already part of a sticky build or no other compatible package version exists.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md:141]:
```suggestion
resolution unless it is already a part of a sticky build or no other compatible package version exists.
```

File: swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md [hunk 2]
```diff
@@ -125,3 +125,24 @@ By default, running the `bal semver` command on the root directory of the packag
...
+If the deprecated version is in use, a warning message containing the provided deprecation message will be shown to the user during project build.
```
Comment (praneesha) [inline swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md:142]:
```suggestion
If the deprecated version is in use, a warning message containing the provided deprecation message will be shown during the project build.
```

Comment (Dilhasha) [reply swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md]: fixed with 2e1a662449fd729658d671fac5c2f4d673582f49

File: swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md (hunk 1 above)
Comment (Dilhasha) [reply swan-lake/learn-the-platform/ballerina-central/publish-packages-to-ballerina-central.md]: fixed with 2e1a662449fd729658d671fac5c2f4d673582f49

**Review by praneesha — APPROVED** [review]

File: swan-lake/learn-the-platform/test-document-the-code/test-ballerina-code/configure-tests.md [hunk 3]
```diff
@@ -273,6 +273,22 @@ function testFunction2() {
 }
 ```
 
+## Ensure test execution order
+
+The test config annotation makes use of ‘dependsOn’ to indicate the specific test cases that the current test relies
```
Comment (azinneera) [inline swan-lake/learn-the-platform/test-document-the-code/test-ballerina-code/configure-tests.md:278]:
```suggestion
The test config annotation makes use of `dependsOn` to indicate the specific test cases that the current test relies 
```

**Review by azinneera — APPROVED** [review]

**Pull request merged**